tests/modules/elmo_test.py

```diff
@@ -97,12 +103,12 @@ def _load_sentences_embeddings(self):
         return sentences, expected_lm_embeddings
 
 
-class TestElmo(AllenNlpTestCase):
+class TestElmo(ElmoTestCase):
     def setUp(self):
         super(TestElmo, self).setUp()
 
-        options_file = os.path.join(FIXTURES, 'options.json')
-        weight_file = os.path.join(FIXTURES, 'lm_weights.hdf5')
+        options_file = os.path.join(self.fixtures_path, 'options.json')
```

Comment:
You could just use the attributes set in the super class here.

Reply:
@DeNeutoy I don't understand.  Can you give an example?

Reply:
Oh just that you're inheriting from `ElmoTestCase` and in `ElmoTestCase.setUp` you set these [variables as attributes](https://github.com/allenai/allennlp/pull/781/files/a2b27a598e1ffd6b8d6a53e765810600d87f1d30#diff-6d5c0b791d43299ce1afe2eda864ec96R25). So instead of re-defining them, you could just use them directly via `self.options_file` etc.

Reply:
Right, thanks.  That's just something I overlooked when I created `ElmoTestCase`.